src/main/resources/beam-template.conf

```diff
@@ -436,6 +436,7 @@ beam.outputs.writeLinkTraversalInterval = "int | 0"
 beam.outputs.writeSkimsInterval = "int | 0"
 beam.physsim.writeEventsInterval = "int | 0"
 beam.physsim.writePlansInterval = "int | 0"
+beam.physsim.writeRouteHistoryInterval = "int | 10"
```

Comment:
Should the default be 10, or 1 for backwards compatibility. Where the bigger thing is that it is now configurable?

Reply:
Hi @JustinPihony this param control routeHistory file writing which is used for warmStart. For warmstart we need around 20 to 30 previous iteration. So we found 10 as default value is appropriate for this param because Sometime user forgot to set this param so its good we write this file in 10,20,30.. iterations